fdbserver/RestoreMaster.actor.cpp

```diff
@@ -690,7 +689,13 @@ ACTOR static Future<Void> collectBackupFiles(Reference<IBackupContainer> bc, std
 		request.targetVersion = desc.maxRestorableVersion.get();
 	}
 
-	Optional<RestorableFileSet> restorable = wait(bc->getRestoreSet(request.targetVersion));
+	if (g_network->isSimulated()) {
+		std::cout << "Restore to version: " << request.targetVersion << "\nBackupDesc: \n" << desc.toString() << "\n\n";
+	}
+
+	Optional<RestorableFileSet> restorable =
+	    wait(SERVER_KNOBS->FASTRESTORE_USE_PARTITIONED_LOGS ? bc->getPartitionedRestoreSet(request.targetVersion)
+	                                                        : bc->getRestoreSet(request.targetVersion));
```

Comment:
I think we can extend the getRestoreSet() parameter list to incorporate the `getPartitionedRestoreSet `.

Again, this is more like refactor instead of correctness.

Reply:
This is intentional to keep existing backup tests intact. I'll consolidate the APIs in a later PR where new partitioned logs become default, and current backups uses the non-default value.